core/container.go

```diff
@@ -804,7 +862,7 @@ func (container *Container) Exec(ctx context.Context, gw bkgw.Client, defaultPla
 	// create /dagger mount point for the shim to write to
 	runOpts = append(runOpts,
 		llb.AddMount(metaMountDestPath,
-			llb.Scratch().File(meta),
+			llb.Scratch().File(meta, llb.WithCustomName("[dagger.internal] creating dagger metadata"), payload.Group.LLB()),
```

Comment:
Looks like this supports taking a progress group, so in this case the CustomName is just to make it clearer in the UI that this is an internal thing?

Reply:
Yes -- this appears as a random `mkdir /meta` and it's not obvious what it stands for (the user requested an exec and this randomly appears)

- `creating dagger metadata` is to make it more obvious than `mkdir /meta`
- `[dagger.internal]` is to hide it altogether from the UI

See comment below for an alternative